lib/src/base-styles/_nx-page-header.scss

```diff
@@ -8,8 +8,8 @@
 @import '../scss-shared/nx-text-helpers';
 @import '../scss-shared/nx-container-helpers';
 
-$main-header-background: $nx-sonatype-blue;
-$main-header-color: $nx-off-white;
+$main-header-background: $nx-off-white;
+$main-header-color: $nx-off-black;
```

Comment:
The text in zeplin seems to use several different colors none of which are this one.

Reply:
> The text in zeplin seems to use several different colors none of which are this one.

Yeah, I talked to Derrek about the colours. We don't have a great solution for that yet so for the text colours I have been using our defaults until he can give us a full palette

Reply:
> Yeah, I talked to Derrek about the colours. We don't have a great solution for that yet so for the text colours I have been using our defaults until he can give us a full palette

I'd rather set it to the right colors through magic numbers now, then when we get variables in place later it's a straightforward search and replace to clean them up.  Rather than setting it to the wrong color now and then having to re-research what colors everything should be when the variables are available.

Reply:
> > Yeah, I talked to Derrek about the colours. We don't have a great solution for that yet so for the text colours I have been using our defaults until he can give us a full palette
> 
> I'd rather set it to the right colors through magic numbers now, then when we get variables in place later it's a straightforward search and replace to clean them up. Rather than setting it to the wrong color now and then having to re-research what colors everything should be when the variables are available.

Colours properly identified here: https://github.com/sonatype/sonatype-react-shared-components/pull/30/commits/f7c7f5512a24c62f07672cbc503805e9e55f2a97